### DISCUSSION TITLE
你们都是几抽出的艾莲？

### DISCUSSION THREAD

**Discussion by sishui-buyin** [share121/inter-knot#829] [其他]
![113829286](https://github.com/user-attachments/assets/d446e564-037e-4f0c-ae0f-8c6e5594cbe1)
你们都是几抽出的艾莲？我不多45抽，但熬了好几天，可以说没有艾莲小姐的日子如地狱

**Comment by ZnS-1145** [comment]
25抽本体，68抽专武

**Comment by Lynette-CN** [comment]
角色小保底，武器大保底。。

**Comment by Kamisato100** [comment]
角色大保底，武器小保底，常驻也是保底，邦布也是保底，太难了

**Comment by 10kaihua** [comment]
角色两小保，武器大保底（2+1）

**Comment by SHENGWANG10086** [comment]
没出=-=

**Comment by ywcSystem** [comment]
没出（大哭

**Comment by gezishiren** [comment]
78抽（悲
